boards/arm/96b_argonkey/doc/96b_argonkey.rst

```diff
@@ -0,0 +1,175 @@
+.. _96b_argonkey:
+
+96Boards Argonkey
+#################
+
+Overview
+********
+
+96Boards Argonkey board is based on the ST Microelectronics STM32F412CG
+Cortex M4 CPU. Zephyr applications use the 96b_argonkey configuration
+to run on these boards.
+
+This board acts as a sensor hub platform for all 96Boards compliant
+family products. It can also be used as a standalone board.
+
+.. figure:: img/96b-argonkey-front.jpg
+     :width: 557px
+     :align: center
+     :height: 480px
+     :alt: 96Boards Argonkey
+
+     96Boards Argonkey
+
+Hardware
+********
+
+96Boards Argonkey provides the following hardware components:
+
+- STM32F412CG in UFQFPN48 package
+- ARM |reg| 32-bit Cortex |reg|-M4 CPU with FPU
+- 100 MHz max CPU frequency
+- 1.8V work voltage
+- 1024 KB Flash
+- 256 KB SRAM
+- On board sensors:
+
+  - Humidity: ST Micro HTS221
+  - Temperature/Pressure: ST Micro LPS22HB
+  - ALS: Intersil ISL29034
+  - Proximity: ST Micro VL53L0X
+  - Acclerometer/Gyroscope: ST Micro LSM6DSL
+  - Geomagnetic: ST Micro LIS2MDL
+  - AMR Hall sensor: MRMS501A
+  - Microphone: ST Micro MP34DT05
+
+- 2 User LEDs
+- 16 General purpose LEDs
+- GPIO with external interrupt capability
+- UART
+- I2C (3)
+- SPI (1)
+- I2S (1)
+
+Supported Features
+==================
+
+The Zephyr 96b_argonkey board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | system clock                        |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash                               |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+
+More details about the board can be found at `96Boards website`_.
```

Comment:
Need to define this link target.

Add:

    .. 96Boards website:  <some url for the target>

Reply:
I fotgot to handle this case properly, but currently we do not have any url for ArgonKey. We are in the process of providing the documentation to 96board. So, again, I will delete this reference and will add it soon after we wiil have the board published on the site.